demo/src/pages/Playground.tsx

```diff
@@ -199,6 +205,52 @@ class Playground extends React.Component<{}, {
       URL.revokeObjectURL(url);
     }
   };
+
+  private _listenAnnotationAdd() {
+    const view3D = this._view3D;
+    const canvas = view3D.renderer.canvas;
+    const raycaster = new THREE.Raycaster();
+
+    canvas.addEventListener("dblclick", evt => {
```

Comment:
I understood if the annotation is added to the location when double-clicking.
Are descriptions of these features provided?

Reply:
![image](https://user-images.githubusercontent.com/26213435/165909696-2064a8b2-f931-4e54-a6c3-59a1d3028963.png)

^ It's provided like the above